Common/Core/RecoDecay.h

```diff
@@ -598,11 +598,11 @@ class RecoDecay
       //printf("Stage %d: Mother PDG: %d\n", stage, PDGParticleIMother);
       if (PDGParticleIMother == PDGMother) { // exact PDG match
         sgn = 1;
-        indexMother = indexMotherTmp;
+        indexMother = particleMother.globalIndex();
```

Comment:
How is `particleMother.globalIndex()` different from `particleMotherIdx`?

Reply:
Indeed, fix it!